### PR TITLE
Remove duplicate point calculate because of consensus and so on

### PR DESCRIPTION
**Pull request by SpriCoder**
Because we need to know the actual external writing point, I try to add `isGenerateByConsensus` parameters in PlanNode and try to remove duplicate points.

### CODE REVIEW

**Comment by SpriCoder** [comment]
Test in IoTConsensus:
image

**Comment by SpriCoder** [comment]
Test in Ratis Consensus:
image

**Comment by SpriCoder** [comment]
Test on 1 node failed:
image

File: iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/ApplicationStateMachineProxy.java [hunk 1]
```diff
@@ -146,6 +146,9 @@ public CompletableFuture<Message> applyTransaction(TransactionContext trx) {
     waitUntilSystemAllowApply();
     do {
       try {
+        if (isLeader) {
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/ApplicationStateMachineProxy.java:149]: !isLeader?

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM~
Awesome work！！！

**Pull request merged**